src/Yoti.Auth/DigitalIdentity/GetSessionResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+
+
+namespace Yoti.Auth.DigitalIdentity
+{
+    public class GetSessionResult
+    {
+#pragma warning disable 0649
+        // These fields are assigned to by JSON deserialization
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        [JsonProperty("status")]
+        public string Status { get; set; }
+
+        [JsonProperty("expiry")]
+        public string Expiry { get; set; }
+
+#pragma warning restore 0649
+
+    }
```

Comment:
Please refer to our internal documentation (sent link in DM)
Depending on the session status you can find a subset of the properties of COMPLETED Session.

If you select the example "Session with status COMPLETED" you will find all the properties we need to map